### PR TITLE
API refactoring.

### PR DESCRIPTION
**Pull request by cemersoz**
Fixes Issues #214, #194

### CODE REVIEW

File: client/imports/ui/pages/lab/labview.ts [hunk 1]
```diff
@@ -110,12 +107,12 @@ export default class LabView extends MeteorComponent {
 
   // Called by Next button
   nextTask(){
-    console.log("proceeding");
     var slf = this;
       Meteor.call('nextTask',"1",function(err,res){
         if(err){
           slf.nextButton = false;
           console.log("try again");
+	  //TODO: @sandershihacker, change this from console.log to alert/something else
```
Comment (DerekTBrown) [inline client/imports/ui/pages/lab/labview.ts]: Why don't you just do it?!!?

File: collections/users.ts [hunk 2]
```diff
@@ -106,7 +106,8 @@ if (Meteor.isServer){
         defaultValue: []
       },
       sessions:{
-        type: sessionSchema
+        type: [sessionSchema],
```
Comment (DerekTBrown) [inline collections/users.ts]: indentation

File: client/imports/ui/pages/lab/labview.ts [hunk 3]
```diff
@@ -110,12 +107,12 @@ export default class LabView extends MeteorComponent {
...
         }
         else{
           console.log(res);
```
Comment (DerekTBrown) [inline client/imports/ui/pages/lab/labview.ts]: Same problem

Comment (cemersoz) [inline client/imports/ui/pages/lab/labview.ts]: Maybe because it's the same line?

**Comment by cemersoz** [comment]
@DerekTBrown I implemented the session-daemon, it wercks.
I made minor cache etcd changes to accommodate...
Please merge this so Sander can work on implementing the front end thing in the future.

**Pull request merged**